tests/unit/sagemaker/image_uris/test_jumpstart.py

```diff
@@ -0,0 +1,110 @@
+
+
+@patch("sagemaker.jumpstart.accessors.JumpStartModelsCache.get_model_specs")
+def test_jumpstart_script_uri(patched_get_model_specs):
+
+    patched_get_model_specs.side_effect = get_spec_from_base_spec
+    uri = image_uris.retrieve(
+        framework=None,
+        region="us-west-2",
+        image_scope="inference",
+        model_id="pytorch-ic-mobilenet-v2",
+        model_version="*",
+        instance_type="ml.p2.xlarge",
+    )
+    assert uri == "763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference:1.5.0-gpu-py3"
+    patched_get_model_specs.assert_called_once_with("us-west-2", "pytorch-ic-mobilenet-v2", "*")
+
+    patched_get_model_specs.reset_mock()
+
+    uri = image_uris.retrieve(
+        framework=None,
+        region="us-west-2",
+        image_scope="training",
+        model_id="pytorch-ic-mobilenet-v2",
+        model_version="*",
+        instance_type="ml.p2.xlarge",
+    )
+    assert uri == "763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.5.0-gpu-py3"
+    patched_get_model_specs.assert_called_once_with("us-west-2", "pytorch-ic-mobilenet-v2", "*")
+    patched_get_model_specs.reset_mock()
+
+    image_uris.retrieve(
+        framework=None,
+        region="us-west-2",
+        image_scope="training",
+        model_id="pytorch-ic-mobilenet-v2",
+        model_version="*",
+        instance_type="ml.p2.xlarge",
+    )
+    patched_get_model_specs.assert_called_once_with(
+        sagemaker_constants.JUMPSTART_DEFAULT_REGION_NAME, "pytorch-ic-mobilenet-v2", "*"
+    )
+
+    with pytest.raises(ValueError):
+        image_uris.retrieve(
+            framework=None,
+            region="us-west-2",
+            image_scope="BAD_SCOPE",
+            model_id="pytorch-ic-mobilenet-v2",
+            model_version="*",
+            instance_type="ml.p2.xlarge",
+        )
+
+    with pytest.raises(ValueError):
+        image_uris.retrieve(
+            framework=None,
+            region="mars-south-1",
+            image_scope="training",
+            model_id="pytorch-ic-mobilenet-v2",
+            model_version="*",
+            instance_type="ml.p2.xlarge",
+        )
+
+    with pytest.raises(ValueError):
+        image_uris.retrieve(
+            framework=None,
+            region="us-west-2",
+            model_id="pytorch-ic-mobilenet-v2",
+            model_version="*",
+            instance_type="ml.p2.xlarge",
+        )
+
+    with pytest.raises(ValueError):
+        image_uris.retrieve(
+            framework=None,
+            region="us-west-2",
+            image_scope="training",
+            model_version="*",
+            instance_type="ml.p2.xlarge",
+        )
+
+    with pytest.raises(ValueError):
+        image_uris.retrieve(
+            region="us-west-2",
+            framework=None,
+            image_scope="training",
+            model_id="pytorch-ic-mobilenet-v2",
+            instance_type="ml.p2.xlarge",
+        )
```

Comment:
Could you please add a unit test for hugging face?

Before we deploy, i would like a unit test for each {framework, scope} we support (which means we will need to break down the tests into separate files)